src/svcutilcore/tests/dotnet-svcutil.xmlserializer.Tests.csproj

```diff
@@ -79,8 +79,13 @@
     <Message Text="Running Serialization Tool" Importance="normal" />
     <PropertyGroup>
       <CorrectedOutputPath>$(OutputPath.Replace('/', '\'))</CorrectedOutputPath>
+      <!--<SMOutputPath>$(CorrectedOutputPath.ToLower().Replace('dotnet-svcutil.xmlserializer.tests', 'system.servicemodel.primitives').Replace('netcoreapp', 'netstandard'))</SMOutputPath>-->
+      <SMOutputPath>$(RuntimePath)</SMOutputPath>
     </PropertyGroup>
-    <Exec Command="$(GeneratorCliPath)dotnet $(CorrectedOutputPath)dotnet-svcutil.xmlserializer.dll $(CorrectedOutputPath)$(InputAssembly).dll /out:$(CorrectedOutputPath)$(SerializerName).cs"  />
+    <Message Text="Run XmlSerializer: $(GeneratorCliPath)dotnet $(CorrectedOutputPath)dotnet-svcutil.xmlserializer.dll $(CorrectedOutputPath)$(InputAssembly).dll /out:$(CorrectedOutputPath)$(SerializerName).cs /smreference:$(SMOutputPath)System.ServiceModel.Primitives.dll" Importance="high" />
+    <Message Text="System.ServiceModel.Primitives.dll exists in $(SMOutputPath)" Condition="Exists('$(SMOutputPath)System.ServiceModel.Primitives.dll') == 'true'" Importance="high" />
+    <Message Text="System.ServiceModel.Primitives.dll not exists in $(SMOutputPath)" Condition="Exists('$(SMOutputPath)System.ServiceModel.Primitives.dll') != 'true'" Importance="high" />
+    <Exec Command="$(GeneratorCliPath)dotnet $(CorrectedOutputPath)dotnet-svcutil.xmlserializer.dll $(CorrectedOutputPath)$(InputAssembly).dll /out:$(CorrectedOutputPath)$(SerializerName).cs /smreference:$(SMOutputPath)System.ServiceModel.Primitives.dll"  />
```

Comment:
Should these messages be converted into an Error so we can fail the test early if S.SM.Primitives isn't where we expect it to be?